components/KiboShippingOptions.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="sf-shipping-options">
+    <slot name="shipping-methods">
+      <SfRadio
+        v-for="(shippingMethod, index) in shippingMethods"
+        :key="`${shippingMethod.name}_${index}`"
+        :name="shippingMethod.name"
+        :value="shippingMethod.value"
+        :label="shippingMethod.label"
+        :details="shippingMethod.details"
+        :required="shippingMethod.required"
+        class="sf-radio"
+        @change="selectShipping($event)"
+      />
+    </slot>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent } from "@vue/composition-api"
+import { SfRadio } from "@storefront-ui/vue"
+
+export default defineComponent({
+  name: "KiboShippingOptions",
+  components: {
+    SfRadio,
+  },
+
+  model: {
+    prop: "shippingMethod",
+  },
+  props: {
+    shippingMethods: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  setup(_, context) {
+    const selectShipping = (selectedMethod: string) => {
+      context.emit("shippingMethod", selectedMethod)
+    }
+
+    return {
+      selectShipping,
+    }
+  },
+})
+</script>
+<style lang="scss">
```

Comment:
Apply Scoped

Reply:
Overriding global CSS for this component so its removed.